dev/tests/integration/testsuite/Magento/Catalog/Model/ProductRepositoryTest.php

```diff
@@ -233,4 +233,24 @@ public function testCustomLayout(): void
         }
         $this->assertTrue($caughtException);
     }
+
+    /**
+     * Tests product repository update should use provided store code.
+     *
+     * @magentoDataFixture Magento/Catalog/_files/product_simple.php
+     */
+    public function testProductUpdate(): void
+    {
+        $sku = 'simple';
```

Comment:
Instead of using local variable - use stubs located in `const`

Reply:
done